proto/stat.proto

```diff
@@ -274,4 +277,16 @@ message GetStatSetRequest {
 message GetStatSetResponse {
   // Keyed by statVar.
   map<string, PlacePointStat> data = 1;
+}
+
+
+message GetPlaceStatDateWithinPlaceRequest {
```

Comment:
nit: I know it makes it long, but I wonder if we should qualify it as WithinAncestorPlace ?

Reply:
here aligns with other "within-place" api names: https://docs.google.com/document/d/12rdNsaLsMj5LxUWS3dynC8fuYTd0sfxqHio0Hh8Joao/edit#heading=h.9ssx4uezxkkc

what do you think?

Reply:
Alright then...